sdk/core/azure-core/azure/core/tracing/ext/opencensus_wrapper.py

```diff
@@ -150,6 +112,9 @@ def set_current_span(cls, span):
         # type: (Span) -> None
         """
         Set the given span as the current span in the execution context.
+
+        :param span: The span to set the current span as
+        :type span: Span
```

Comment:
And this not being a built-in type would probably be `:class:opencensus.trace.Span` (whatever the fully-qualified name is). (The cross-ref link probably doesn't work in this case, but that's the syntax)

Reply:
done